altair/vegalite/v5/api.py

```diff
@@ -1055,6 +1055,25 @@ def to_html(
             requirejs=requirejs,
         )
 
+    def to_url(self, fullscreen: bool = False) -> str:
```

Comment:
Consider: forcing all args to be keyword-only so that in the future, if we add more params to this function, they are not order dependent and we can re-order them as needed without breaking anyone. Not sure if this is overkill. 

to_url(False) is also not very obvious what False does. Requiring a kwarg improves that.

On the other hand, requiring a kwarg is more typing.

I think I fall 60% in favor of requiring kwarg but am fine with keeping it as is

Reply:
Good idea! Done in [5d440f9](https://github.com/altair-viz/altair/pull/3252/commits/5d440f98a9cceadd88cb51fa4d1f78556a067b0a)